src/DevBetterWeb.Infrastructure/Services/DailyCheckPingService.cs

```diff
@@ -64,11 +68,49 @@ public async Task SendPingIfNeeded(AppendOnlyStringList messages)
       }
     }
 
+    public async Task PingAdminsAboutAlmostAlumsIfNeeded(AppendOnlyStringList messages)
+    {
+      var members = await _memberRepository.ListAsync();
+      var membersToPingAdminsAbout = new List<Member>();
+
+      var daysBeforeGraduationToPing = 10;
```

Comment:
Maybe pull out to a constant as well. Eventually these things might come from a config file.